[INTERNAL] GettingStarted: Update minimum node version, Workspace: Add details, Migrate v3: Describe CLI changes (#759)
SAP/ui5-tooling: docs/pages/Overview.md

## Linking Projects
Would you like to work on an application project and one or more of its dependencies at the same time? We got you covered!

### UI5 Workspaces
The recommended approach for setting up a development environment where simultaneous work on multiple UI5 projects is required.

> Head over to the [UI5 Workspaces](./Workspace.md) documentation for all the details. Below is an example based on a simple scenario with an application and a reuse library project:

Reviewer: ```suggestion
Head over to the [UI5 Workspaces](./Workspace.md) documentation for the details. Below is an example based on a simple scenario with an application and a reuse library project:
```

**Example: Your Directory Structure**
```
my-app/
    \_ node_modules/
    \_ webapp/
    \_ ui5.yaml
    \_ package.json
my-reuse-library/
    \_ node_modules/
    \_ src/
    \_ test/
    \_ ui5.yaml
    \_ package.json
```

> In its `package.json`, `my-app` should already define a dependency to `my-reuse-library`. So, after running the `npm install` command, a copy of the "my-reuse-library"-package should be retrieved from the package registry and added to my-app's `node_modules/` directory.

Reviewer 2: ```suggestion
In its `package.json`, `my-app` should already define a dependency to `my-reuse-library`. So, after running the `npm install` command, a copy of the `my-reuse-library`-package should be retrieved from the package registry and added to my-app's `node_modules/` directory.
```

> In its `package.json`, `my-app` should already define a dependency to `my-reuse-library`. So, after running the `npm install` command, a copy of the "my-reuse-library"-package should be retrieved from the package registry and added to my-app's `node_modules/` directory.

Reviewer 2: should we add here a disclaimer regarding `npm install` in npm version 9 to add `--install-links false`?

Author: Thanks! But that option is actually only required when working with the `file://` protocol in dependencies (e.h. `"my-reuse-library": "file://../my-reuse-library"`. And even in that case, symlinks instead of copies are perfect since we can then omit the `npm link` step.

In this example however, we expect `my-reuse-library` to be a normal dependency stored in an npm registry. So npm install will just fetch a copy and store it in `node_modules/`. This behavior did not change with npm 9

In the `my-app` directory, create a new file named `ui5-workspace.yaml` with the following content:

```yaml title="ui5-workspace.yaml"
specVersion: workspace/1.0
metadata:
    name: default
dependencyManagement:
    resolutions:
        - path: ../my-reuse-library
```

> That's it! Start a server using `ui5 serve` and you will see any changes you make in `my-reuse-library` are immediately visible in `my-app`.

Reviewer: ```suggestion
That's it! Start a server via `ui5 serve`, and you will see that any changes you make in `my-reuse-library` are immediately visible in `my-app`.
```

## Package Managers

> By leaving dependency management up to the tool of your choice (see [FAQ: Why package.json? / Why npm?](./FAQ.md#why-packagejson-why-npm)) you have a variety of other options for linking dependencies into your project..

Reviewer: ```suggestion
By leaving dependency management up to the tool of your choice (see [FAQ: Why package.json? / Why npm?](./FAQ.md#why-packagejson-why-npm)) you have a variety of other options for linking dependencies into your project.
```

> Here is an example using the [npm CLI](https://www.npmjs.com/get-npm) in version 8, an application, and a reuse library project:

Reviewer: ```suggestion
Here is an example using the [npm CLI](https://www.npmjs.com/get-npm) in Version 8, an application, and a reuse library project:
```
